src/server/pages/home-page.jsx

```diff
@@ -24,15 +29,67 @@ const LetsGetStarted = () => (
 
 export default class HomePage extends React.Component {
   render() {
+    const isLoggedIn = 'name' in this.props.userInfo
     const { notebookList } = this.props
     return (
       <div>
         <Header userInfo={this.props.userInfo} />
         <PageBody>
-          {notebookList.length ?
-            <TrendingNotebooksPage notebookList={notebookList} /> :
-            <LetsGetStarted />
+          <TopContainer>
+            {!isLoggedIn && <MarketingCopySplash /> }
```

Comment:
We probably don't want to show the marketing splash for the internal server. 

For the legal splash we use a variable called `IODIDE_PUBLIC_TOS`. Maybe we could rename it to `IODIDE_PUBLIC` and use that here? https://github.com/iodide-project/iodide/blob/master/webpack.config.js#L27

Reply:
Fixed in a recent commit.